src/main/java/se/lth/cs/connect/routes/Admin.java

```diff
@@ -88,6 +90,52 @@ protected void setup(PippoSettings conf) {
             });
             rc.getResponse().ok();
         });
+        
+        
+        GET("/collections-owned-by", (rc)->{
+        	if(rc.getParameter("email").isEmpty())
+        		throw new RequestException("must provide an user email parameter");
+
+        	final JcNode coll = new JcNode("coll");
+        	final JcString cString = new JcString("cString");
+        	final JcQueryResult res = Database.query(rc.getLocal("db"), new IClause[]{
+        		MATCH.node().label("user").property("email").value(rc.getParameter("email").toString())
+        		.relation().type("OWNER").node(coll).label("collection"),
+        		RETURN.value(coll.property("name")).AS(cString)
+        	});
+        	
+            final List<String> found = res.resultOf(cString);
+            String[] colls = new String[found.size()];
+            found.toArray(colls);
+            rc.status(200).json().send(colls);
+        	
+        });
+        
+        // POST api.serp.se/v1/admin/delete-user
+        // email=...
+        POST("/delete-user", (rc) -> {
+        	if(rc.getParameter("email").isEmpty())
+        		throw new RequestException("must provide an user email parameter");
+
+        	AccountSystem.deleteAccount(rc.getParameter("email").toString(),rc.getLocal("db"));
+        	rc.getResponse().ok();
```

Comment:
We should probably return some error if the request fails. Or is that taken care of in the deleteAccount function?